test/compflags/ferguson/unstable-class.good

```diff
@@ -1,5 +1,3 @@
-unstable-class.chpl:20: warning: undecorated class type MyClass is unstable
-unstable-class.chpl:20: note: use 'unmanaged MyClass' 'owned MyClass', 'borrowed MyClass', or 'shared MyClass'
```

Comment:
Is the above warning gone for a good reason?

Reply:
It's not gone, just printed in a different order (I think).